### PR TITLE
first change

### PR DESCRIPTION
**Pull request by dianaeldemerdash**

### CODE REVIEW

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        LIMS-4330
+        :return:
+        """
+        self.order_page.get_random_orders()
+        order_url = self.base_selenium.get_url()
+        current_no = self.order_page.get_no()
+        new_no = self.generate_random_string()
+        self.order_page.set_no(new_no)
+        if 'save_btn' == save:
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: when could be happened?

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            self.order_page.save(save_btn='order:save')
+        else:
+            self.order_page.cancel(force=True)
+
+        self.base_selenium.get(url=order_url, sleep=self.base_selenium.TIME_X_LARGE)
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: Too long sleep time

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        if 'save_btn' == save:
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: when could be happened?

**Comment by dianaeldemerdash** [comment]
about this one @Islam Taha <islam.taha@modeso.ch>

**Pull request closed without merging**